charts/komodor-agent/values.yaml

```diff
@@ -343,3 +343,4 @@ communications:
   wsHost: wss://app.komodor.com
   tasksServerHost: *serverHost
   telemetryServerHost: https://telemetry.komodor.com
+  mgmtServerHost: *serverHost
```

Comment:
I don't see we use this value in the template....

Reply:
we copy everything `communications` to config,yaml
![Uploading image.png…]()

Reply:
Yes, thanks @nirbenator